zoo-project/zoo-services/utils/open-api/dru/DeployProcess.py

```diff
@@ -141,16 +141,16 @@ def get_application_package(self):
             raise ValueError("The inputs dot not include applicationPackage")
 
         # loading cwl in yaml object
-        if "cache_file" in self.inputs["applicationPackage"]:
-            logger.info(
-                f"Loading CWL from cache file {self.inputs['applicationPackage']['cache_file']}"
-            )
-            cwl_content = yaml.safe_load(
-                open(self.inputs["applicationPackage"]["cache_file"]).read()
-            )
-        else:
-            logger.info("Loading CWL from value")
-            cwl_content = yaml.safe_load(self.inputs["applicationPackage"]["value"])
+        # if "cache_file" in self.inputs["applicationPackage"]:
+        #     logger.info(
+        #         f"Loading CWL from cache file {self.inputs['applicationPackage']['cache_file']}"
+        #     )
+        #     cwl_content = yaml.safe_load(
+        #         open(self.inputs["applicationPackage"]["cache_file"]).read()
+        #     )
+        # else:
```

Comment:
Is it better to remove completely for maintaining code quality, or might we want to reinstate this soon? We can recover it from version control if needed

Reply:
Agreed, removed commented code